packages/babel-parser/src/tokenizer/types.js

```diff
@@ -140,6 +140,7 @@ export const types: { [name: string]: TokenType } = {
 
   eq: new TokenType("=", { beforeExpr, isAssign }),
   assign: new TokenType("_=", { beforeExpr, isAssign }),
+  slashAssign: new TokenType("_=", { beforeExpr, isAssign }),
```

Comment:
nit
```suggestion
  slashAssign: new TokenType("/=", { beforeExpr, isAssign }),
```

Reply:
The token type label is visible when `tokens: true` is enabled so it may breaks if people are depending on `token.label` since `/=` was parsed as `tt.assign` whose label is `/=`. We can modify the label in Babel 8.

Reply:
why wrap it in a new class and use the 'new' keyword. Did you run a benchmark test on that?

Reply:
The token is initiated once and reused elsewhere. I am aware that using a binary packed number is more optimal because it eliminates the memory loading instructions on compiled from `tt.slashAssign`. However this PR is refactoring, it only tries to not degrade the performance so reviewers can be focused on the code architecture changes. We will land performance improvement in a separate PR.